single_group_playbooks/repo.yml

```diff
@@ -7,9 +7,10 @@
   roles:
     - admin_users
     - ssh_host_signer
+    - iptables
+    - yum_repos
     - ssh_known_hosts
     - swap
-    - {role: geerlingguy.repo-epel, become: true}
     - {role: yum_repos, when: repo_manager == 'none'}
     - logrotate
     - logins
```

Comment:
I here missing `epel` role?

Reply:
NVM I see that epel is defined in yum_repos...

but the question then is, why did you create `single_role_playbooks/epel.yml`?